### PR TITLE
Sentry Integration

### PR DESCRIPTION
**Pull request by deepak1725**
# Description

Added Sentry Logs integration. 
The developer now can use an instance to log errors into Sentry.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: recommendation_engine/flask_predict.py [hunk 1]
```diff
@@ -76,5 +83,13 @@ def recommendation():
     return flask.jsonify(response_json), 200
 
 
+def log_it(func):
```
Comment (miteshvp) [inline recommendation_engine/flask_predict.py]: Where is this decorator used? I do not see it being used

File: requirements.in [hunk 2]
```diff
@@ -5,3 +5,4 @@ gevent==1.2.2
 numpy==1.14.2
 scipy==1.0.0
 gunicorn==19.7.1
+raven[flask]
```
Comment (miteshvp) [inline requirements.in]: Add a blank line after this statement

File: recommendation_engine/flask_predict.py (hunk 1 above)
Comment (deepak1725) [reply recommendation_engine/flask_predict.py]: future reference

Comment (miteshvp) [reply recommendation_engine/flask_predict.py]: What future reference are you planning?

**Pull request merged**